react-app/cypress/integration/search_page.spec.js

```diff
@@ -0,0 +1,55 @@
+/// <reference types="cypress" />
+
+context('Movies page', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:9001')
+    cy.wait(2000)
```

Comment:
That kind of `cy.wait()` use may be considered a sign of the *magic number* [test smell](https://testsmells.org/pages/testsmells.html#MagicNumberTest). Furthermore, it is [discouraged](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting) by Cypress set of best practices. It may cause your tests either to wait (which may become quite an issue when used within the pipeline job with the big number of tests) unnecessarily (when page is loaded before 2s timeout is over) or to fail because of some episodic network lag (which delays page load longer than 2s), making them flaky. You might want to have a look at some [examples](https://docs.cypress.io/api/commands/wait#Examples) of using `cy.wait()` properly.

Reply:
All tests run properly without wait. Thanks for the advice.